jlink.go

```diff
@@ -0,0 +1,200 @@
+package libjvm
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/buildpacks/libcnb"
+	"github.com/heroku/color"
+	"github.com/magiconair/properties"
+	"github.com/paketo-buildpacks/libjvm/count"
+	"github.com/paketo-buildpacks/libpak"
+	"github.com/paketo-buildpacks/libpak/bard"
+	"github.com/paketo-buildpacks/libpak/effect"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+)
+
+type JLink struct {
+	LayerContributor  libpak.LayerContributor
+	Logger            bard.Logger
+	ApplicationPath   string
+	Executor          effect.Executor
+	CertificateLoader CertificateLoader
+	Metadata          map[string]interface{}
+	JavaVersion       string
+	Args              []string
+	UserConfigured    bool
+}
+
+func NewJLink(applicationPath string, exec effect.Executor, args []string, certificateLoader CertificateLoader, metadata map[string]interface{}, userConfigured bool) (JLink, error) {
+	expected := map[string]interface{}{"jlink-args": args}
+	if md, err := certificateLoader.Metadata(); err != nil {
+		return JLink{}, fmt.Errorf("unable to generate certificate loader metadata\n%w", err)
+	} else {
+		for k, v := range md {
+			expected[k] = v
+		}
+	}
+	contributor := libpak.NewLayerContributor(
+		"jlink",
```

Comment:
Very minor. All of our layer names seem to start with capital letters except for this new layer.

```
  Microsoft OpenJDK 11.0.15: Reusing cached layer
  jlink: Contributing to layer
   ....
  Launch Helper: Reusing cached layer
  Java Security Properties: Reusing cached layer
```

It's kind of tricky cause `jlink` is usually seen in lower case. What about `Java Runtime with jlink` or just `JLink` for the layer name? or something else?